README.md

```diff
@@ -37,25 +37,25 @@ stock_item::{ // stock item has no name
 """
 
 val ion = IonSystemBuilder.standard().build
-val stockItems = ion.newReader(stockItemsIonText).use { reader ->
+val stockItems = ION.newReader(stockItemsIonText).use { reader ->
```

Comment:
Is it really `ION`?

Reply:
oh, yeah, I will lowercase that real quick